src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+.page__body {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0;
+
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.page__form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-form::before {
+  content: "";
+  display: block;
+  width: 19px;
+  height: 19px;
+  background-image: url(./images/Search.svg);
+  background-position: center;
+  background-size: cover;
+  position: absolute;
+  top: 26px;
+  left: 26px;
+}
+
+.search-form--size--small::before {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
```

Comment:
let's not hard code height for input.
just use proper paddings

Reply:
Got it! Thanks!